DemoApp/DemoAppCoordinator.swift

```diff
@@ -227,7 +249,17 @@ final class DemoAppCoordinator: NSObject, UNUserNotificationCenterDelegate {
     }
     
     static func logout(window: UIWindow) {
+        let currentUserController = ChatClient.shared.currentUserController()
+        if let deviceId = currentUserController.currentUser?.currentDevice?.id {
+            currentUserController.removeDevice(id: deviceId) { error in
+                if let error = error {
+                    log.warning("removing the device failed with an error \(error)")
+                }
+            }
+        }
         ChatClient.shared.disconnect()
+        UserDefaults(suiteName: applicationGroupIdentifier)?
```

Comment:
With this change, we start repeating ourselves. How about:
```swift
extension UserDefaults {
    static let shared = UserDefaults(suiteName: applicationGroupIdentifier)
}
```

Reply:
makes sense 👍